### PR TITLE
Reconfigure codecov

### PR DESCRIPTION
**Pull request by rossabaker**
Bonus: this one is actually valid.

### CODE REVIEW

**Comment by bryce-anderson** [comment]
👍

**Comment by rossabaker** [comment]
I can't reproduce this [Travis failure](https://gist.github.com/rossabaker/eab11cea4228c3ab31fe66833ef6307a) locally.  This is purely a codecov change, and it's not even one of the builds that does codecov.

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=0.13.12
```
Comment (rossabaker) [inline project/build.properties]: This is not germane to the ticket, but is a good thing to do, and I'm hoping it jiggles this stubborn Travis failure.

Comment (rossabaker) [inline project/build.properties]: That fixed it.  We got the intermittent client failure again, but not the persistent sbt crash.

**Pull request merged**